frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -475,134 +478,109 @@ class Chart extends Component<Props, State> {
     const toolTipTop = Math.floor(Math.max(coordinateY, 0));
     const toolTipLeft = Math.floor(Math.max(40, Math.min(parent.clientWidth - 140, point.x + 40 - 70)));
 
-    this.setState({
+    setTooltipData({
       toolTipVisible: true,
-      toolTipValue: this.formattedData ? this.formattedData[time as number] : '',
+      toolTipValue: formattedData.current ? formattedData.current[time as number] : '',
       toolTipTop,
       toolTipLeft,
       toolTipTime: time as number,
     });
-  };
+  }
 
-  private renderDate = (date: number) => {
-    return new Date(date).toLocaleString(
-      this.props.i18n.language,
-      {
-        year: 'numeric',
-        month: '2-digit',
-        day: '2-digit',
-        ...(this.state.source === 'hourly' ? {
-          hour: '2-digit',
-          minute: '2-digit',
-        } : null)
-      }
-    );
+  const {
+    lastTimestamp,
+    chartDataMissing,
+    chartFiat,
+    chartIsUpToDate,
+    chartTotal,
+    formattedChartTotal,
+  } = data;
+
+
+  const {
+    toolTipVisible,
+    toolTipValue,
+    toolTipTop,
+    toolTipLeft,
+    toolTipTime,
+  } = tooltipData;
+
+
```

Comment:
nit: for readability and consistency there is no need to have multiple newlines, here and before